### PR TITLE
Initial commit for the new SF end point

### PR DESCRIPTION
**Pull request by himaninare**

### CODE REVIEW

**Review by amrutha-sireesh — CHANGES_REQUESTED** [review]

File: src/main/java/com/ccs/conclave/api/cii/data/OrgDataProvider.java [hunk 1]
```diff
@@ -123,6 +122,19 @@ public static SchemeInfo getExpSchemeInfoWithOnlySFIdentifier(SchemeRegistry sch
         return schemeInfoModified;
     }
 
+    public static String getOnlySFIdentifier(SchemeRegistry schemeRegistry) {
```
Comment (amrutha-sireesh) [inline src/main/java/com/ccs/conclave/api/cii/data/OrgDataProvider.java:125]: This method is no longer used.

File: src/main/java/com/ccs/conclave/api/cii/verification/VerifyEndpointResponses.java [hunk 2]
```diff
@@ -233,6 +247,31 @@ public static void verifyRegisteredSchemes(Response actualRes, SchemeInfo expect
         }
     }
 
+    public static void verifySFRegisteredSchemesAsAdditionalIdentifiers(Response actualRes, SchemeInfo expectedSchemeInfo, int selectedAddIds) {
```
Comment (amrutha-sireesh) [inline src/main/java/com/ccs/conclave/api/cii/verification/VerifyEndpointResponses.java:250]: Is there any difference between this method and verifyRegisteredSchemes()?

**Review by amrutha-sireesh — CHANGES_REQUESTED** [review]

File: src/main/java/com/ccs/conclave/api/cii/data/OrgDataProvider.java (hunk 1 above)
Comment (himaninare) [reply src/main/java/com/ccs/conclave/api/cii/data/OrgDataProvider.java:125]: Removed

File: src/main/java/com/ccs/conclave/api/cii/verification/VerifyEndpointResponses.java (hunk 2 above)
Comment (himaninare) [reply src/main/java/com/ccs/conclave/api/cii/verification/VerifyEndpointResponses.java:250]: verifyRegisteredSchemes checks for the Primary Identifiers along with additional identifiers whereas the verifySFRegisteredSchemesAsAdditionalIdentifiers checks for additional identifiers only thats y i created one more method

**Review by amrutha-sireesh — APPROVED** [review]

**Pull request merged**